manager/controllers/template_systems_delete.go

```diff
@@ -34,13 +34,15 @@ func TemplateSystemsDeleteHandler(c *gin.Context) {
 
 	db := middlewares.DBFromContext(c)
 
-	// unassign system from template => assign NULL as template_id
-	err := assignTemplateSystems(c, db, account, nil, req.Systems, groups)
+	err := checkTemplateSystems(c, db, account, nil, req.Systems, groups)
 	if err != nil {
 		return
 	}
 
-	err = assignCandlepinEnvironment(db, account, nil, req.Systems, groups)
+	modified, _ := assignCandlepinEnvironment(db, account, nil, req.Systems, groups)
```

Comment:
why is the error not checked here? If the error is never checked we might want to ignore errors also in the `assignCandlepinEnvironment` function and not return error